src/main/java/uk/gov/hmcts/sscs/deserialize/CcdResponseDeserializer.java

```diff
@@ -339,22 +338,22 @@ public void deserializeEvidenceDetailsJson(JsonNode caseNode, CcdResponse ccdRes
         if (evidenceNode != null) {
             final JsonNode documentsNode = evidenceNode.get("documents");
```

Comment:
There is a better way to handle this. Done by @sathishbabuj you can find examples in case loader -> https://github.com/hmcts/sscs-case-loader/blob/master/src/main/java/uk/gov/hmcts/reform/sscs/util/CcdUtil.java#L14 

Nice to have.